cmd/docs/command.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2019 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package docs
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/cobra/doc"
+)
+
+// Command creates `docs` command
+func Command() *cobra.Command {
+	return &cobra.Command{
+		Use:   "docs",
+		Short: "Generates documentation",
+		Long:  `Generates command and flags documentation`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			for cmd.Parent() != nil {
+				cmd = cmd.Parent()
+			}
+			return doc.GenMarkdownTree(cmd, "./")
```

Comment:
ran this locally, it's kind of misleading that there's no output from the command. I would recommend at least some log file "generating .md files in the current directory"